test/travis.deps.bash

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#
+# This script installs and configures the dependencies for the sbt-extras
+# Author: Kevin Brightwell (Github: Nava2)
+
+declare OS_NAME
+
+case `uname` in 
+    Darwin)     OS_NAME="osx"    ;;
+    Linux)      OS_NAME="linux"  ;;
+esac
+
+if env | grep -qE '^(?:TRAVIS|CI)='; then
+#    We're on Travis, intialize variables:
+    echo "Detected CI Build -> CI=${CI}"
+
+else
+#   We're building locally
+    echo "This script is meant to be run by Travis-CI services."
+
+    # exit 1
+fi
+
+echo "Building on: ${OS_NAME}"
+
+install_os_deps() {
+    # Install all of the OS specific OS dependencies
+    local JDK_VERSION_LEVEL=${JDK_VERSION:(-1)}
+
+    echo "Attempting to use Java: ${JDK_VERSION}"
+
+    case ${OS_NAME} in
+        osx)
+            if [[ ${JDK_VERSION} == openjdk* ]] ; then
+                echo "OpenJDK is not supported on OSX, defaulting to oraclejdk${JDK_VERSION_LEVEL}"
+                export JDK_VERSION=oraclejdk${JDK_VERSION_LEVEL}
+            fi
+            
+            # Install the correct version of Java:
+
+            case ${JDK_VERSION_LEVEL} in
+                8)
+                    CASK_PKG=caskroom/cask;
+                    JAVA_PKG="java"
+                    ;;
+                7|9)
+                    CASK_PKG=caskroom/versions;
+                    JAVA_PKG="java${JDK_VERSION_LEVEL}";
+                    #echo "Invalid Java Version, ${JDK_VERSION}"
+                    # exit 1
```

Comment:
I'll clean-up the comments, sorry I missed them. 

For installing Java 7, the version on the system by default is 7u45, this will install 7u80 as shown [here](https://travis-ci.org/paulp/sbt-extras/jobs/103668091#L89). It seems like a good idea to keep it up-to-date, but I'll let you be the final judge.